cmd/root_options.go

```diff
@@ -132,11 +132,21 @@ func (ro *RootOptions) toBuild() *builder.Build {
 		KernelUrls:       ro.KernelUrls,
 		RepoOrg:          ro.Repo.Org,
 		RepoName:         ro.Repo.Name,
+		Images:           make(builder.ImagesMap),
+	}
+
+	if len(build.BuilderRepos) == 0 {
```

Comment:
I'd inizialize it as a default value during declaration, using the `default:` tag; we use this library: https://github.com/creasty/defaults#usage

As you can see, a 
```
BuilderRepos []string `default: ["docker.io/falcosecurity/driverkit"] validate:"omitempty" name:"docker repositories to look for builder images or absolute path pointing to a file container builder image index"`
```
Is enough.

Reply:
Oh I didn't know we could place complex types inside this default statement